src/tram/tram/models.py

```diff
@@ -88,11 +88,12 @@ class DocumentProcessingJob(models.Model):
     updated_on = models.DateTimeField(auto_now=True)
 
     @classmethod
-    def create_from_file(cls, f):
+    def create_from_file(cls, f, u):
```

Comment:
IMO, we should start adding docstrings on any classes/methods we modify that don't already have them.

Reply:
Added docstrings for this function. What do you think?